### ISSUE TITLE
App crashes on the Ti.Geolocation.reverseGeocoder

### ISSUE THREAD

**Issue by thomasheijmans** [appcelerator-developer-relations/Sample.Mapping#38]
Running the app clean out of the box and it exits immediately. I had this issue in my own app and tried to run this one to check if it had the same issue.

    //Geocoder.app/alloy/controllers/index.js

    function reverseGeocode(coords, center) {
      'use strict';

      // Don't re-use coords since reverseGeocode() is also a callback for two
      // events in the view, which has other properties as well that we don't need.
      var location = {
        latitude: coords.latitude,
        longitude: coords.longitude
      };

      // Reverse geocode the position
      Ti.Geolocation.reverseGeocoder(location.latitude, location.longitude, function(e) {

    if (!e.success || e.error) {
      return alert(e.error || 'Could not reverse geocode the position.');
    }

    // Use the address of the first place found for the title
    location.title = e.places[0].address;

    // Drop or move the annotation
    setAnnotation(location);

    // center the map on the annotation
    if (center) {
      centerMap(location);
    }
      });
    }

[ERROR] :  Script Error {
[ERROR] :      column = 35;
[ERROR] :      line = 154;
[ERROR] :      message = "*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[5]";
[ERROR] :      nativeStack = (
[ERROR] :  	0   CoreFoundation                      0x00007fff23b98bde __exceptionPreprocess + 350
[ERROR] :  	1   libobjc.A.dylib                     0x00007fff503b5b20 objc_exception_throw + 48
[ERROR] :  	2   CoreFoundation                      0x00007fff23c2af71 _CFThrowFormattedException + 194
[ERROR] :  	3   CoreFoundation                      0x00007fff23c35926 -[__NSPlaceholderDictionary initWithObjects:forKeys:count:].cold.4 + 38
[ERROR] :  	4   CoreFoundation                      0x00007fff23bf6417 -[__NSPlaceholderDictionary initWithObjects:forKeys:count:] + 247
[ERROR] :  	5   CoreFoundation                      0x00007fff23b94a8b +[NSDictionary dictionaryWithObjects:forKeys:count:] + 59
[ERROR] :  	6   Geocoder                            0x0000000102921eca -[GeolocationModule performGeo:address:callback:] + 394
[ERROR] :  	7   Geocoder                            0x0000000102921fd4 -[GeolocationModule reverseGeocoder:longitude:withCallback:] + 180
[ERROR] :  	8   CoreFoundation                      0x00007fff23b9f95c __invoking___ + 140
[ERROR] :  	9   CoreFoundation                      0x00007fff23b9cd8f -[NSInvocation invoke] + 287
[ERROR] :  	10  JavaScriptCore                      0x00007fff269674bb _ZN3JSC24ObjCCallbackFunctionImpl4callEP9JSContextP13OpaqueJSValuemPKPKS3_PS6_ + 411
[ERROR] :  	11  JavaScriptCore                      0x00007fff26966f04 _ZN3JSCL34objCCallbackFunctionCallAsFunctionEPK15OpaqueJSContextP13OpaqueJSValueS4_mPKPKS3_PS6_ + 244
[ERROR] :  	12  JavaScriptCore                      0x00007fff2696682f _ZN3JSC19APICallbackFunction4callINS_20ObjCCallbackFunctionEEExPNS_9ExecStateE + 511
[ERROR] :  	13  ???                                 0x000022e50e001027 0x0 + 38367177740327
[ERROR] :  	14  JavaScriptCore                      0x00007fff2693caa3 llint_entry + 93226
[ERROR] :  	15  JavaScriptCore                      0x00007fff2693caa3 llint_entry + 93226
[ERROR] :  	16  JavaScriptCore                      0x00007fff26925ccf vmEntryToJavaScript + 200
[ERROR] :  	17  JavaScriptCore                      0x00007fff26d361a0 _ZN3JSC11Interpreter11executeCallEPNS_9ExecStateEPNS_8JSObjectENS_8CallTypeERKNS_8CallDataENS_7JSValueERKNS_7ArgListE + 416
[ERROR] :  	18  JavaScriptCore                      0x00007fff26f82e24 _ZN3JSC12profiledCallEPNS_9ExecStateENS_15ProfilingReasonENS_7JSValueENS_8CallTypeERKNS_8CallDataES3_RKNS_7ArgListE + 196
[ERROR] :  	19  JavaScriptCore                      0x00007fff26983328 JSObjectCallAsFunction + 488
[ERROR] :  	20  JavaScriptCore                      0x00007fff26950d1c -[JSValue callWithArguments:] + 252
[ERROR] :  	21  TitaniumKit                         0x0000000102c6f293 -[ObjcProxy _fireEventToListener:withObject:listener:] + 287
[ERROR] :  	22  Geocoder                            0x0000000102925138 -[GeolocationModule fireSingleShotLocationIfNeeded:stopIfNeeded:] + 456
[ERROR] :  	23  Geocoder                            0x0000000102925f1c -[GeolocationModule locationManager:didUpdateLocations:] + 284
[ERROR] :  	24  CoreLocation                        0x00007fff24787561 CLClientStopVehicleHeadingUpdates + 70612
[ERROR] :  	25  CoreLocation                        0x00007fff24786cbb CLClientStopVehicleHeadingUpdates + 68398
[ERROR] :  	26  CoreLocation                        0x00007fff24771c2a CLClientInvalidate + 1437
[ERROR] :  	27  CoreFoundation                      0x00007fff23afb8ec __CFRUNLOOP_IS_CALLING_OUT_TO_A_BLOCK__ + 12
[ERROR] :  	28  CoreFoundation                      0x00007fff23afb058 __CFRunLoopDoBlocks + 312
[ERROR] :  	29  CoreFoundation                      0x00007fff23af646b __CFRunLoopRun + 2699
[ERROR] :  	30  CoreFoundation                      0x00007fff23af56b6 CFRunLoopRunSpecific + 438
[ERROR] :  	31  GraphicsServices                    0x00007fff3815cbb0 GSEventRunModal + 65
[ERROR] :  	32  UIKitCore                           0x00007fff47162a67 UIApplicationMain + 1621
[ERROR] :  	33  Geocoder                            0x00000001027525c2 main + 1042
[ERROR] :  	34  libdyld.dylib                       0x00007fff5123bcf5 start + 1
[ERROR] :  );
[ERROR] :      sourceURL = "file:///Users/thomas/Library/Developer/CoreSimulator/Devices/EC3AFD03-3933-48DD-87CC-38DA965D104A/data/Containers/Bundle/Application/ED98B755-9713-4895-BAB3-A83FB25C7626/Geocoder.app/alloy/controllers/index.js";
[ERROR] :      stack = "[native code]\nreverseGeocode@file:///Users/thomas/Library/Developer/CoreSimulator/Devices/EC3AFD03-3933-48DD-87CC-38DA965D104A/data/Containers/Bundle/Application/ED98B755-9713-4895-BAB3-A83FB25C7626/Geocoder.app/alloy/controllers/index.js:154:35\nfile:///Users/thomas/Library/Developer/CoreSimulator/Devices/EC3AFD03-3933-48DD-87CC-38DA965D104A/data/Containers/Bundle/Application/ED98B755-9713-4895-BAB3-A83FB25C7626/Geocoder.app/alloy/controllers/index.js:130:23";
[ERROR] :  }

**Comment by antonioduran** [comment]
Hello. Good afternoon / evening.

I am getting the same results. Did you find any workaround @thomasheijmans ?

Thanks, and best,
Antonio Duran.

**Comment by thomasheijmans** [comment]
Yes, by using an external API (here.com) that returns the location, simply just not using the module. No response from the Appcelerator people either.
